06/ukol11.py

```diff
@@ -0,0 +1,17 @@
+#Napiš funkci tah_hrace, která dostane řetězec s herním polem, zeptá se hráče, na kterou pozici chce hrát,
+#a vrátí herní pole se zaznamenaným tahem hráče.
+#Funkce by měla odmítnout záporná nebo příliš velká čísla a tahy na obsazená políčka. Pokud uživatel
+#zadá špatný vstup, funkce mu vynadá a zeptá se znova.
+
+def tah_hrace(field):
+    '''Vrati herni pole se zaznamenanym tahem hrace.'''
+    while True:
+        player_position = int(input('Zadej cislo pozice: '))
```

Comment:
A trochu škoda, že když tady zadám třeba "ahojky", tak mi program celý spadne 😄 Mohlo by to ještě přes starý známý `str.isdigit()` kontrolovat, zda se fakt zadá číslo - a pokud ne, zeptá se znovu.